services/response_code.proto

```diff
@@ -1530,18 +1530,28 @@ enum ResponseCodeEnum {
      */
     UPDATE_NODE_ACCOUNT_NOT_ALLOWED = 359;
 
+    /*
+     * NFT already exist in the pending state.
+     */
+    PENDING_NFT_AIRDROP_ALREADY_EXISTS = 360;
```

Comment:
One new response is added in this PR https://github.com/hashgraph/hedera-protobufs/pull/394. We should move the response code numbers here

Reply:
Thanks, will do!